examples/atlas/log-download.md

```diff
@@ -0,0 +1,9 @@
+# Download and Unzip Logs
+
+You can download and unzip logs in one go like
```

Comment:
I think you mean line here but I may be wrong
```suggestion
You can download and unzip logs in one line like
```

Reply:
but it's not one line, it's three

Reply:
yes, you are right xD forget my comment 🙄